packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -310,7 +311,10 @@ export const HelpCenterContactForm = () => {
 			section: sectionName,
 		} );
 
+		const savedCurrentSite = currentSite;
 		resetStore();
+		setSite( savedCurrentSite );
+
 		navigate( '/' );
```

Comment:
Maybe we should instead just call `setShowHelpCenter( false )` to close the Help Center since the user should be done?